pulseapi/entries/views.py

```diff
@@ -126,11 +128,19 @@ class EntryView(RetrieveAPIView):
     """
     A view to retrieve individual entries
     """
-    queryset = Entry.objects.public()
+
+    #queryset = Entry.objects.public()
+
+    def get_queryset(self):
```

Comment:
we did not have this in place. I don't know what the impact of that was before, but it's kind of important to filter on moz auth or not now!